tests/org.jboss.tools.jst.ui.bot.test/pom.xml

```diff
@@ -38,4 +40,42 @@
 		</plugins>
 	</build>
 
+	<profiles>
+		<profile>
+			<id>check-pr</id>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.eclipse.tycho</groupId>
+						<artifactId>target-platform-configuration</artifactId>
+						<configuration>
+							<systemProperties>
+								<org.osgi.framework.bundle.parent>ext</org.osgi.framework.bundle.parent>
+							</systemProperties>
+							<dependency-resolution>
+								<extraRequirements>
+									<requirement>
+										<type>p2-installable-unit</type>
```

Comment:
Are you going to build this feature locally ? If not, than there is no point specifying it as installable unit